ci/aws/aws-cluster.tf.envsubst

```diff
@@ -0,0 +1,21 @@
+module "aws-tempest" {
+  source = "../../aws/flatcar-linux/kubernetes"
+
+  providers = {
+    aws = "aws.default"
+    local = "local.default"
+    null = "null.default"
+    template = "template.default"
+    tls = "tls.default"
+  }
+
+  cluster_name = "$CLUSTER_ID"
+  dns_zone = "$AWS_DNS_ZONE"
+  dns_zone_id = "$AWS_DNS_ZONE_ID"
+  ssh_authorized_key = "$PUB_KEY"
+
+  asset_dir = "${pathexpand("~/lokomotive-assets")}"
+
+  worker_count = 2
+  worker_type  = "t3.small"
```

Comment:
In Packet we are defining the controllers (number and size) and that matches the default. I thought we wanted to be explicit, just in case the default changes or something.

Should we add the controllers size and count here, too? Or remove it on Packet?

IMHO, it makes sense to be explicit and specify them, so tests don't depend on the value of some defaults and we test exactly what we want to test